typedocReadMe.md

```diff
@@ -18,15 +46,92 @@ The study metadata defines elements such as the study title, authors, and descri
 The UI configuration tells reVISit how the UI should be laid out, such as which image to use for the study logo, whether to include a sidebar, the contact email, etc. For more detailed documentation on the UI configuration, check out the [documentation](/typedoc/interfaces/UIConfig.html).
 
 
-## Study Components
+## Components
+
+Components are the building blocks for each study. 
+There are currently 5 types of components: 
+* [Image](/typedoc/interfaces/ImageComponent.html), 
+* [Website](/typedoc/interfaces/WebsiteComponent.html), 
+* [Questionnaire](/typedoc/interfaces/QuestionnaireComponent.html), 
+* [React](/typedoc/interfaces/ReactComponent.html), and 
+* [Markdown](/typedoc/interfaces/MarkdownComponent.html).
+
+Each component extends the [BaseIndividualComponent](/typedoc/interfaces/BaseIndividualComponent.html) interface. To add a component to your study (which can be thought of as a "page" of your study), you add a JSON object representing that component to the "components" object with a key which you can define how you would like. Then, the `type` key in that JSON object controls which type of component you are referring to. 
+
+
+### Collecting Responses
+
+Each component has a list of responses which represents a set of questions to ask to the user for that particular component. The user can describe where the question should be displayed in the UI, the type of response input (e.g., a [numerical response](/typedoc/interfaces/NumericalResponse.html), a [dropdown](/typedoc/interfaces/DropdownResponse.html), a [slider](/typedoc/interfaces/SliderResponse.html), and more. Each response interface extends the [BaseResponse](/typedoc/interfaces/BaseResponse.html) interface. 
+
+The below example illustrates a simple consent component that is based on a Markdown file and has a response that asks for a signature: 
+
+```JSON
+  "consent": {
+            "type": "markdown",
+            "path": "demo-brush-interactions/assets/consent.md",
+            "nextButtonText": "Agree",
+            "response": [
+                {
+                    "id": "signature",
+                    "prompt": "Your signature",
+                    "required": true,
+                    "location": "belowStimulus",
+                    "type": "shortText",
+                    "placeholder": "Please provide your signature"
+                }
+            ]
+        }
+```
+
+For more detailed documentation on the response section, check out the [documentation](/typedoc/index.html#response).
+
+## Base Components and Inheritance
+
+[Base Components](/typedoc/interfaces/StudyConfig.html#basecomponents] can be used to implement inheritance for components. This is often useful if you want to parameterize a component. For example: 
```

Comment:
```suggestion
[Base Components](/typedoc/interfaces/StudyConfig.html#basecomponents) can be used to implement inheritance for components. This is often useful if you want to parameterize a component. For example: 
```

Reply:
Fixed in [97bb812](https://github.com/revisit-studies/study/pull/344/commits/97bb81206e28892a4322a791dc2d3c14fe868155)